dashboard/src/components/AppView/AppView.tsx

```diff
@@ -103,6 +106,7 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
     const deployments = manifest.filter(d => d.kind === "Deployment");
```

Comment:
I think that eventually we should refactor this, it looks quite inefficient. 

It seems that the initial filter could be done by mapping all the resources by kind and then for the sockets initialization just iterate on the map based on a whitelist of resources.